src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/SqlBulkCopyTest/DataConversionErrorMessageTest.cs

```diff
@@ -0,0 +1,169 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Data;
+using System.Text;
+using Xunit;
+
+namespace Microsoft.Data.SqlClient.ManualTesting.Tests
+{
+    internal enum ColumnsEnum
+    {
+        _int = 0,
+        _varChar3 = 1
+    }
+
+    public class InitialDatabase : IDisposable
+    {
+        private string srcConstr { get; }
+
+        public SqlConnection Connection { get; }
+
+        public string TableName { get; }
+
+        public InitialDatabase()
+        {
+            srcConstr = DataTestUtility.TCPConnectionString;
+
+            Connection = new SqlConnection(srcConstr);
+            TableName = $"SqlBulkCopyTest_CopyStringToIntTest_{Guid.NewGuid().ToString().Replace('-', '_')}";
```

Comment:
You can use DataTestUtility.GetUniqueNameForSqlServer to generate table names for tests.